### PR TITLE
Terms and Conditions bar

### PR DESCRIPTION
**Pull request by BurgundyIsAPublicEnemy**
# Issue Being Addressed
201

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For Bug Fixes:** What was the root cause? How do your changes fix the bug effectively?

**For Refactors:** What are we refactoring and why? How do your changes address this need for refactoring?

**For New Features:** What feature are we adding? Explain your technical approach to doing this.
Adding a bottom tool bar for seeing terms and conditions and agreeing to them. A new component (termsandconditions) is called in the homepage (this can be done in other pages too if need be) which has a link to terms and conditions as well as a button to accept the terms and conditions

**For Feature Updates:** What feature are we updating? Explain your technical approach to doing this.

**For Other:** State what 'type' of PR yours is and elaborate on the PR's technical content.

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Run foodouken as normal
2. Scroll the bottom.
3. Check if bar is viewable

# Screenshots/casts
image

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

**Comment by bamboochow** [comment]
Thank you for contributing! 

# Snackbar Location
This snackbar should be sticky to the bottom of the viewing window. It is currently sticky to the bottom of the page. 
It should remain in the same scrolling position as users are scrolling down the page 

# Mobile Alignment 
There is a misalignment of the snackbar on mobile, where the text and snackbar position is overextending past the viewing window and requires a scroll. The text is also too close to the text button. 
we follow https://material.io/components/snackbars#usage in snackbar position and usage https://material.io/develop/android/components/snackbars/

image

Ideal spacing for mobile 
image

**Comment by atharva3010** [comment]
First of all, thank you for your contribution 👍. A few things to note:

- We use "2 spaces" indentation for our code. Our Prettier configuration enforces that, please make sure you have the Prettier plugin for VSCode installed.
- We TitleCase our component names.
- The project uses TailwindCSS, it'd be a good idea to use it instead of writing CSS from scratch.

File: src/components/termsandconditions/termsconditions.vue [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        this.showaccept();
+      }
+    },
+    updateValue(option) {
+      this.$emit('updateselectedVisability', option);
+      this.hideaccept();
+    }
+  }
+};
+</script>
```
Comment (atharva3010) [inline src/components/termsandconditions/termsconditions.vue:149]: - Please do not waste efforts writing css from scratch. These properties are included as utilities in TailwindCSS which we use throughout our projects. It has predefined color, text and spacing utilities used throughout the project.
- Components in this project follow the standard Vue SFC declaration pattern, i.e.
```js
<template>
...
</template>

<script>
...
</script>

<style>
...
</style>
```
Though if you use tailwind's utilities, you won't need the style block.
- For naming the component, please use TitleCase, eg: TermsAndConditionBar.vue
- Please use Prettier and ESLint VSCode plugin (if you use VSCode) and follow it's recommendations while coding. For safety, please run `yarn lint` before you commit your changes.
Our project uses "2 space" indentation.

**Comment by paulchrisluke** [comment]
this still needs to be done, so keeping it open and moving to backlog

**Pull request closed without merging**